docs/Local Development Install.md

```diff
@@ -0,0 +1,181 @@
+Table of Contents
+
+1. [DCMFOSS Local Development Deployment](#dcmfoss-local-development-deployment)
+   - [Prerequisites](#prerequisites)
+   - [Guide](#guide)
+      - [Cloning the repository](#cloning-the-repository)
+      - [Open in your own IDE of choice](#open-in-your-own-ide-of-choice)
+      - [Docker desktop](#docker-desktop)
+      - [Running containers](#running-containers)
+      - [Running project first time for init configs](#running-project-first-time-for-init-configs)
+      - [Running Keycloak](#running-keycloak)
+      - [Fetching the keycloak client credential](#fetching-the-keycloak-client-credential)
+      - [Configure postman collection requests](#configure-postman-collection-requests)
+      - [Run the front-end](#run-the-front-end)
+   - [Postmand Collection](#Postman-collection)
+   - [Notice](#notice)
+
+
+# DCMFOSS Local Development Deployment
+
+
+## Prerequisites
+
+This guide assumes the following prerequisites are true.
+It was built considering an updated windows build with docker running, but will try to provide general instructions.
+
+  - Basic coding knowledge
+  - Basic Docker/Kubernetes knowledge
+  - JDK 17 ( or Higher)
+  - Maven 'Spring Boot is compatible with Apache Maven 3.5 or above'
+  - Git/Github Desktop
+  - NodeJS installed
+  - Yarn installed
+  - Docker Desktop
+
+
+## Guide:
+Let's begin local development install!
+
+- ### Cloning the repository
+    Clone the repository from the url below
+
+      git clone https://github.com/catenax-ng/tx-demand-capacity-mgmt.git
```

Comment:
wrong URL

Reply:
Fixed via commit:
https://github.com/catenax-ng/tx-demand-capacity-mgmt/commit/ede3b3ac66c05c6c0c1ef5acef34c1df97927f6e

Reply:
@ClosedSourcerer can you please re-check your fix? You just altered the org name, but did not adjust the repo name, so every Link now results in 404

Reply:
@SebastianBezold 
Fixed via a different commit.

Reply:
Sorry, missed, that the repo name was also fixed in a later commit